src/core/software/pupil-player.md

```diff
@@ -37,6 +37,19 @@ The Player window is the main control center for `Pupil Player`. It displays vid
 1. **Menu**: This area contains settings and contextual information for each plugin.
 1. **Sidebar**: This area contains clickable buttons for each plugin. System plugins are loaded in the top and user added plugins are added below the horizontal separator.
 
+### Keyboard Shortcuts
+
+| Keyboard Shortcut   | Description                                      |
+|:--------------------|:-------------------------------------------------|
+| `<space>`           | Play and pause video                             |
+| `<arrow left>`      | Step to previous frame / Decrease playback speed |
```

Comment:
can you clarify how this keyboard shortcut can be both stepping and increasing/decreasing playback speed?

Reply:
The frame skipping and playback-speed changes are explained directly above the table. Do you think this is too much out of context and requires to be re-iterated?

Reply:
What do you think about this footnote solution?
![Screenshot 2021-04-13 at 09 18 58](https://user-images.githubusercontent.com/168390/114512333-4931c700-9c39-11eb-9441-a1809c4fd832.png)